tests/test_report.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from . import v
+assert v  # Silence pyflakes
+
+mock_code = """\
+import foo
+
+class Foo:
+    '''
+    This is a long class
+    '''
+    def __init__(self):
+        print("Initialized foo")
+
+    def bar(self):
+        self.foobar = "unused attribute"
+        foobar = "unused variable"
+        return
+        print("A small unused function")
```

Comment:
This is not a function. Let's just use `print("unreachable")`.

Reply:
oops! :-p